test/explain/basic.py

```diff
@@ -324,9 +324,15 @@ def WriteInitialTest( valueDict ) :
 python_sep = python.replace('\\', '\\\\')
 
 expect = test.wrap_stdout("""\
-scons: rebuilding `file3' because the dependency order changed:
-               old: ['xxx', 'yyy', 'zzz', '%(python_sep)s']
-               new: ['zzz', 'yyy', 'xxx', '%(python_sep)s']
+scons: rebuilding `file3' because:
+           the dependency order changed:
+           ->Sources
+           Old:xxx	New:zzz
+           Old:yyy	New:yyy
+           Old:zzz	New:xxx
+           ->Depends
+           ->Implicit
+           Old:/usr/bin/python	New:/usr/bin/python
```

Comment:
That explicit string as a Python path won't work on Windows.

Reply:
What do you mean?  I'm just dumping the bimplicit's..

Reply:
See the appveyor build:  /usr/bin/python is not what it returns:

```
38/300 (12.67%) c:\\python27\\python.exe -tt test\explain\basic.py
STDOUT =========================================================================
12c12
<            Old:/usr/bin/python	New:/usr/bin/python
---
>            Old:C:\python27\python.exe	New:C:\python27\python.exe
FAILED test of C:\projects\scons\src\script\scons.py
	at line 615 of C:\projects\scons\testing\framework\TestCommon.py (_complete)
```

Reply:
Fixed.